.github/workflows/ci.yml

```diff
@@ -1,9 +1,10 @@
-name: CI
+name: "Test CI"
+
 on:
   push:
-    branches: [ master ]
```

Comment:
let's bring that back?

Reply:
Contributors can't use the action on other branches when forked with it.
I don't exactly know why, but it shouldn't harm to leave that out?